logging/cluster-logging-release-notes.adoc

```diff
@@ -1,18 +1,62 @@
 [id="cluster-logging-release-notes"]
-= Release notes for Red Hat OpenShift Logging 5.2
+= Release notes for Red Hat OpenShift Logging 5.3
 include::modules/common-attributes.adoc[]
 :context: cluster-logging-release-notes-v5x
 
 toc::[]
 
-[id="openshift-logging-supported-versions"]
-== Supported versions
-
-OpenShift Logging 5.0, 5.1, and 5.2 run on {product-title} 4.7 and 4.8.
-
 include::modules/con_making-open-source-more-inclusive.adoc[leveloffset=+1]
 
+[id="cluster-logging-supported-versions"]
+== Supported Versions
+include::modules/cluster-logging-supported-versions.adoc[leveloffset=+1]
+
 // Release Notes by version
+[id="cluster-logging-release-notes-5-3-0"]
+== OpenShift Logging 5.3.0
+The following advisories are available for OpenShift Logging 5.3.x:
+
+[id="openshift-logging-5-3-0-new-features-and-enhancements"]
+=== New features and enhancements
+* With this update, authorization requirements for Log Forwarding have been relaxed. Outputs may now be configured with SASL, username/password, or TLS.
```

Comment:
I assume the update relaxes the Log Forwarding requirement? I agree that putting it in the passive voice decreases the anthropomorphism (as opposed to saying `This update relaxes the authorization requirements`. 

What do you think about this rewording that eliminates the anthropomorphism all together?

`With this update, there are less authorization requirements for Log Forwarding.` or `With this update, there are less strict/rigorous/rigid requirements for Log Forwarding.`

Reply:
What about this:  `With this update, Log Forwarding has less strict authorization requirements. You can now configure outputs using SASL, username/password, or TLS.`

However, that being said, I guess I'm not really sure what `relaxed` means in the original context. Does it mean fewer? or less strict (if so, how, specifically?)? or does it mean that people with lower levels of permissions can now log in? It's not clear to me.

Also the second sentence doesn't follow from the first. These are just additional authentication methods, not necessarily less strict ones. It sounds like these additional methods are another enhancement rather than an extension of the first sentence.

Reply:
How about... "With this update, Log Forwarding does not require mTLS. You can now configure outputs to authenticate using SASL, username/password, or TLS."

Reply:
@libander it support mTLS before. Just doesn't support SASL.

Reply:
Before update Log Forwarding only support  plaintext and TLS.  after this update.  Log Forwarding support SASL over mechanisms PLAIN(username/password). You can enable both Plaintext + SASL and SSL + SASL

Reply:
@anpingli - Prior to the update, log forwarding forced mTLS - this allows for TLS / username password / SASL. Related PR: https://github.com/openshift/openshift-docs/pull/37798